### PR TITLE
Added materia colorscheme

### PR DESCRIPTION
**Pull request by lokesh-krishna**
I edited the existing `one` colorscheme and replaced all of the colors with appropriate colors from [base16-materia](https://github.com/Defman21/base16-materia-scheme).

Did it so that it fits in with the base16-materia vim colorscheme that I use and hope that it might be of use to someone else as well.

### CODE REVIEW

File: doc/lightline.txt [hunk 1]
```diff
@@ -229,7 +229,7 @@ OPTIONS						*lightline-option*
 		Currently, wombat, solarized, powerline, jellybeans, Tomorrow,
 		Tomorrow_Night, Tomorrow_Night_Blue, Tomorrow_Night_Eighties,
 		PaperColor, seoul256, landscape, one, Dracula, darcula,
-		Molokai, OldHope and 16color are available.
+		Molokai, ,materia, OldHope and 16color are available.
```
Comment (itchyny) [inline doc/lightline.txt]: [nit] `, ,` should be `,`

**Comment by itchyny** [comment]
I could not find a `materia` colorscheme for Vim. Can you find a repository for it? Instead I found [vim-material-theme](https://github.com/jdkanani/vim-material-theme), which seems to be much more famous. If the colors matches with the `vim-material-theme`, I think the name should be `material`.

**Comment by lokesh-krishna** [comment]
Here is the colorscheme I am using: [base16-materia](https://github.com/chriskempson/base16-vim/blob/master/colors/base16-materia.vim). 

The colors match this theme and so I feel that materia or base16-materia is what this should be called.

**Comment by itchyny** [comment]
Okay, I prefer renaming to `base16-materia` then.

File: autoload/lightline/colorscheme/materia.vim [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+let s:p = {'normal': {}, 'inactive': {}, 'insert': {}, 'replace': {}, 'visual': {}, 'tabline': {}}
+
+if &background ==# 'light'
+  " Light variant
+  let s:bg     = '#ffffff'
+  let s:gray1  = '#2c393f'
+  let s:gray2  = '#d5dbe5'
+  let s:gray3  = '#707880'
+
+  let s:p.normal.left     = [ [ s:bg, s:green, 'bold' ], [ s:gray1, s:gray3 ] ]
```
Comment (itchyny) [inline autoload/lightline/colorscheme/materia.vim]: It's not a strict convention but I prefer blue for normal and green for insert.

Comment (lokesh-krishna) [reply autoload/lightline/colorscheme/materia.vim]: Just noticed that that is true for almost all of the offered colorschemes but I edited the one theme and the colors are swapped in it :P 

I've swapped them back now :)

**Comment by itchyny** [comment]
Thank you.

**Pull request merged**